server/routes/additional-learning-needs-screener/create/check-your-answers/checkYourAnswersController.test.ts

```diff
@@ -7,14 +7,17 @@ import { aValidAlnScreenerDto } from '../../../../testsupport/alnScreenerDtoTest
 import ChallengeType from '../../../../enums/challengeType'
 import StrengthType from '../../../../enums/strengthType'
 import AdditionalLearningNeedsScreenerService from '../../../../services/additionalLearningNeedsScreenerService'
+import AuditService from '../../../../services/auditService'
```

Comment:
Whilst this looks like a big PR, most of the changes are the same.
We are basically wiring in 7 event types, so for each of the 7 types the controller has the audit service injected and the relevant audit service method calls. This accounts for 3 files per event type - the controller, the controller unit test, and the route definition. Which equates to 21 files

The other 2 files that have been changed (👇 ) cater for the addition of the new event type for refused/declined ELSPs